utils/common.ts

```
@@ -0,0 +1,442 @@
+/**
+ * Formats a number as a localized currency string.
+ *
+ * Returns "Price not specified" if the input price is null or undefined. Uses "KES" as the default currency.
+ *
+ * @param price - The numeric value to format as currency
+ * @param currency - The ISO currency code to use (default is "KES")
+ * @returns The formatted currency string or a default message if price is not provided
+ */
+export function formatPrice(
+  price: number | null | undefined,
+  currency: string = "KES",
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)